geometry_msgs/Vector3[] path
---
float64 minDist
